### ISSUE TITLE
yarn: does not work anymore with any nodejs version besides 10

### ISSUE THREAD

**Issue by Johnnynator** [void-linux/void-packages#21953]
### System

* xuname:  
  *x86_64{,-musl}*
* package:  
  *yarn-1.22.4_1 nodejs-13.2.0_1 nodejs-lts-12.13.1_1*

### Expected behavior
yarn install to give me my node_modules dir, like it did 5d ago
### Actual behavior
yarn install to exits with 0 even so it stopped in step [2/4]
```
yarn install --verbose
yarn install v1.22.4
verbose 0.093521179 Checking for configuration file "/builddir/electron8-8.2.5/electron-8.2.5/.npmrc".
verbose 0.093702241 Checking for configuration file "/tmp/.npmrc".
verbose 0.093816811 Checking for configuration file "/usr/etc/npmrc".
verbose 0.093905812 Checking for configuration file "/builddir/electron8-8.2.5/electron-8.2.5/.npmrc".
verbose 0.094008343 Checking for configuration file "/builddir/electron8-8.2.5/.npmrc".
verbose 0.094103354 Checking for configuration file "/builddir/.npmrc".
verbose 0.094463056 Checking for configuration file "/builddir/electron8-8.2.5/electron-8.2.5/.yarnrc".
verbose 0.094556647 Checking for configuration file "/tmp/.yarnrc".
verbose 0.094640577 Checking for configuration file "/usr/etc/yarnrc".
verbose 0.094720408 Checking for configuration file "/builddir/electron8-8.2.5/electron-8.2.5/.yarnrc".
verbose 0.094799049 Checking for configuration file "/builddir/electron8-8.2.5/.yarnrc".
verbose 0.094878799 Checking for configuration file "/builddir/.yarnrc".
verbose 0.096710562 current time: 2020-05-14T11:03:54.367Z
$ node -e 'process.exit(0)'
verbose 0.176363564 Performing "GET" request to "https://yarnpkg.com/latest-version".
[1/4] Resolving packages...
[2/4] Fetching packages...
verbose 0.451804106 Performing "GET" request to "https://registry.yarnpkg.com/@types/multiparty/-/multiparty-0.0.32.tgz".
verbose 0.45512447 Performing "GET" request to "https://registry.yarnpkg.com/@electron/docs-parser/-/docs-parser-0.4.2.tgz".
verbose 0.456046196 Performing "GET" request to "https://registry.yarnpkg.com/chalk/-/chalk-2.4.2.tgz".
verbose 0.456942262 Performing "GET" request to "https://registry.yarnpkg.com/@types/temp/-/temp-0.8.34.tgz".
verbose 0.459259109 Performing "GET" request to "https://registry.yarnpkg.com/@types/node/-/node-12.6.1.tgz".
verbose 0.460140205 Performing "GET" request to "https://registry.yarnpkg.com/@types/markdown-it/-/markdown-it-0.0.7.tgz".
verbose 0.46090724 Performing "GET" request to "https://registry.yarnpkg.com/chai/-/chai-4.2.0.tgz".
```

### Steps to reproduce the behavior
E.g. `./xbps-src pkg chronograft` doesn't build anymore.

**Comment by pepe** [comment]
It seems http is not working.

**Comment by Johnnynator** [comment]
Yeah. The problem was the llhttp bump. https://github.com/void-linux/void-packages/commit/406bb901b9dd5f61b37e3df657ec08baa7492c2d fixed it for nodejs, I'm currently test #21886 which should resolve this also for nodejs-lts.

**Comment by Johnnynator** [comment]
9bba07ff0cb8b2c851e73c1c377e6646b86af658 should have resolved this for nodejs-lts.
